google-cloud-bigtable/acceptance/bigtable_helper.rb

```diff
@@ -191,6 +196,10 @@ def clean_up_bigtable_objects instance_id, table_ids = []
 $bigtable_read_table_id = "r-#{Date.today.strftime "%y%m%d"}-#{SecureRandom.hex(2)}"
 $bigtable_mutation_table_id = "r-#{Date.today.strftime "%y%m%d"}-#{SecureRandom.hex(2)}"
 
+# Allow overriding the KMS key used for tests via an environment variable. These keys are public, but access may be
+# restricted when tests are run from a VPC project.
+$kms_key = ENV["BIGTABLE_TEST_KMS_KEY"] || "projects/helical-zone-771/locations/us-east1/keyRings/bigtable-test/cryptoKeys/bigtable-test-1"
```

Comment:
Any way we can not have so many global variables? I worry about there being collisions across tests when we run CI jobs that encompass tests across multiple libraries.

Reply:
Sure, I'll try to remove most of them. It would be great to clean this up finally, thanks for suggesting it.

Reply:
I removed about half of the globals from both the acceptance and the samples helpers.

Reply:
Yeah we don't have to change all of them now. I was just hoping not to create any more of them. :)